hip-service-test/Link/Patient/LinkPatientRepositoryTest.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using hip_service.Link.Patient;
+using hip_service.Link.Patient.Models;
+using hip_service_test.Link.Builder;
+using Microsoft.EntityFrameworkCore;
+using Xunit;
+
+namespace hip_service_test.Link.Patient
+{
+    [Collection("Link Patient Repository Tests")]
+    public class LinkPatientRepositoryTest
+    {
+        public LinkPatientContext GetLinkPatientContext()
+        {
+            var optionsBuilder = new DbContextOptionsBuilder<LinkPatientContext>()
+                .UseInMemoryDatabase("hipservice")
+                .Options;
+            return new LinkPatientContext(optionsBuilder);
+        }
+        
+        [Fact]
+        private async void  ShouldReturnSaveLinkRequest()
```

Comment:
should return what? and linkPatientRepository.SaveLinkPatientDetails this can be simpler. :)